### PR TITLE
Api documentation

### PR DESCRIPTION
**Pull request by cverna**
DRF has by default a nice built in doc, so lets use that instead of the django-rest-swagger extension

### CODE REVIEW

File: fpdc/urls.py [hunk 1]
```diff
@@ -30,5 +29,5 @@
     url(r"^admin/", admin.site.urls),
     url(r"^oidc/", include("mozilla_django_oidc.urls")),
     url(r"^api/v1/", include((router.urls, "releases"), namespace="v1")),
-    url(r"^$", schema_view),
```
Comment (abompard) [inline fpdc/urls.py:33]: So there's no landing page anymore? Does the coreapi-based client still work if there's nothing on `/`? IIRC that's where it gets the CoreAPI schema, by requesting a specific content-type.

Comment (cverna) [reply fpdc/urls.py:33]: Ha I need to double check, the plan was to redirect `/` to `/docs` on the proxies

Comment (cverna) [reply fpdc/urls.py:33]: > So there's no landing page anymore? Does the coreapi-based client still work if there's nothing on `/`? IIRC that's where it gets the CoreAPI schema, by requesting a specific content-type.

I have updated the PR to use DRF schema generator on `/`, I have also added the possibility to set the URL used in the API schema we need this in prod and stg to make sure that we use `https`.

**Review by abompard — APPROVED** [review]
Looks nice, thanks! :-)

**Pull request merged**